ceph-releases/ALL/centos/daemon-base/__ISCSI_PACKAGES__

```diff
@@ -1 +0,0 @@
-tcmu-runner ceph-iscsi python-rtslib
```

Comment:
FYI: `ceph-iscsi` is now available on chacra [1], `python3-rtslib` is part of CentOS 8 [2]. I'll get a development build for `tcmu-runner` under el8 on chacra available tomorrow.

[1] https://2.chacra.ceph.com/r/ceph-iscsi/master/cc2aeb4013225b25135ec6e5fb454c7329fb9772/centos/8/flavors/default/noarch/ceph-iscsi-3.3-14.gcc2aeb4.el8.noarch.rpm
[2] http://mirror.centos.org/centos/8/BaseOS/x86_64/os/Packages/python3-rtslib-2.1.fb69-3.el8.noarch.rpm

Reply:
thanks for the quick update !

Reply:
Where does tcmu-runner come from?

Reply:
Yeah. I was planning to kick off the el8 build once we had the el8 ceph build available since it depends on it.

Reply:
I'll add the ceph-iscsi and tcmu-runner packages back in a separate PR